packages/app-content-pages/package.json

```
@@ -16,6 +17,7 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@next/bundle-analyzer": "~13.4.13",
```

This installs the bundle analyser for Next 13.  The package version might need to match our NextJS version.

---

```suggestion
    "@next/bundle-analyzer": "~12.3.0",
```

This is worth trying, if the bundle analyzer was available in version 12.